test/wdio/stencil-sibling/cmp.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { h } from '@stencil/core';
+import { render } from '@wdio/browser-runner/stencil';
+
+describe('stencil-sibling', function () {
+  beforeEach(() => {
+    render({
+      template: () => <stencil-sibling></stencil-sibling>,
+    });
+  });
+
+  it('loads sibling root', async () => {
+    const stencilSibling = await $('stencil-sibling');
+    await stencilSibling.waitForStable();
```

Comment:
Non-blocking - does this test animate? Could this be
```suggestion
    await stencilSibling.waitForExist();
```

Reply:
oh yup, looks like that works 👍